examples/spatial_transformer_network.ipynb

```diff
@@ -0,0 +1,518 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Using DNN layers\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Using gpu device 0: Graphics Device\n"
+     ]
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "import os\n",
+    "os.environ['THEANO_FLAGS']='device=gpu0'\n",
+    "\n",
+    "import matplotlib\n",
+    "import numpy as np\n",
+    "np.random.seed(123)\n",
+    "import matplotlib.pyplot as plt\n",
+    "import lasagne\n",
+    "import theano\n",
+    "import theano.tensor as T\n",
+    "try:\n",
+    "    from lasagne.layers import dnn  # fails early if not available\n",
+    "    conv = dnn.Conv2DDNNLayer\n",
+    "    pool = dnn.MaxPool2DDNNLayer\n",
+    "    print \"Using DNN layers\"\n",
+    "except:\n",
+    "    conv = lasagne.layers.Conv2DLayer\n",
+    "    pool = lasagne.layers.MaxPool2DLayer\n",
+    "    print \"DNN not available, using standard (slower) conv and pool layers\"\n",
```

Comment:
That's a myth, though. If you use the standard layers, the speed should be exactly the same, as Theano will use cuDNN both for convolution and pooling if it's available. It only makes a difference if you use strided pooling / convolution, because Theano may introduce a suboptimal gradient then (this will be fixed in future). Can you remove those checks to not spread the myth? (Again, it may be easiest to edit the file by hand. Note that you will also have to remove the `Using DNN layers\n` output above in this case.)